data/templates/install-ubuntu/ubuntu-preseed

```diff
@@ -195,4 +195,6 @@ in-target wget http://<%=server%>:<%=port%>/api/current/templates/post-install-u
 in-target chmod +x ./post-install.sh; \
 in-target ./post-install.sh; \
 in-target wget http://<%=server%>:<%=port%>/api/current/templates/ubuntu-sources -O /etc/apt/sources.list; \
+in-target wget http://<%=server%>:<%=port%>/api/current/templates/<%=rackhdCallbackScript%> -O /etc/rc2.d/S99callback; \
```

Comment:
rc2.d is specific to run level, all file under rc2.d is a symlink to ./init.d. maybe it could be optimized here?